file2.js

```diff
@@ -0,0 +1,558 @@
+/* global QamineUtils, Stripe */
+
+"use strict";
+
+(function($, window) {
+  var classVar = (function() {
```

Comment:
![Codacy](https://app.dev.codacy.org/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function. (wrap-iife)](https://app.dev.codacy.org/app/codacytestuser/CheckPrCommentsPublicProject/pullRequest?prid=7992)